.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
+    labels:
+      - ci
+      - github-actions
```

Comment:
I know I am the dumb a$$ who added these, but could you add `- dependencies`?
I should have done so too, and will fix it for the jf-sdk

Reply:
```suggestion
      - ci
      - github-actions
      - dependencies
```